server/src/test/java/org/elasticsearch/search/SearchModuleTests.java

```diff
@@ -303,7 +304,7 @@ public void testRegisterRescorer() {
             "combined_fields",
             "dis_max",
             "exists",
-            "field_masking_span",
+            NAME.getPreferredName(),
```

Comment:
Let's just use "span_field_masking" here to keep in line with the other entries in the list.